hostcall/libdevice/src/hostcall_stubs.hip

```diff
@@ -74,15 +73,15 @@ EXTERN uint32_t __strlen_max(char *instr, uint32_t maxstrlen) {
 //
 
 EXTERN int vector_product_zeros(int N, int*A,  int*B, int*C) {
-  ulong arg0,arg1,arg2,arg3,arg4,arg5,arg6,arg7;
-  arg0 = (long) N;
+  uint64_t arg0,arg1,arg2,arg3,arg4,arg5,arg6,arg7;
+  arg0 = (int64_t) N;
   // Pass these pointers to the host for memcpy
-  arg1 = (long) A;
-  arg2 = (long) B;
-  arg3 = (long) C;
+  arg1 = (int64_t) A;
+  arg2 = (int64_t) B;
+  arg3 = (int64_t) C;
   hostcall_result_t result = hostcall_invoke(PACK_VERS(HOSTCALL_SERVICE_DEMO),
     arg0,arg1,arg2,arg3,arg4,arg5,arg6,arg7);
-  int rc = (int) result.arg0;
+  (int) result.arg0;
```

Comment:
What does this line do? Do you want to just remove this line?

Reply:
Nothing. The function is essentially documentation, so I think it's telling the reader that arg0 is a field in hostcall_result_t. Happy to put the assignment back in, clang warned about the unused variable.

Reply:
You don't have to put the assignment back in. Just that a non-assignment seems odd. No strong opinions either ways.

Reply:
Works for me. Deleted the statement on the latest revision.